bundles/org.openhab.binding.freemobilesms/src/main/java/fr/free/smsapi/Account.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package fr.free.smsapi;
```

Comment:
Why keep this in a separate package ?

Reply:
I picked this code from an other project of mine.
So I kept it as separated code.

Perhaps I can rewrite it, but I would appreciate to have an initial version accepted in OpenHAB, if acceptable.

Reply:
From my point of view, the footprint of the external project if very low in this (especially once you'll use org.eclipse.smarthome.io.net.http.HttpUtil.executeUrl ), so I really do not see the interest of keeping this separate package.

Reply:
Yes please put the classes in the standard binding package structure.